src/main/scala/castalia/Receptionist.scala

```diff
@@ -6,6 +6,8 @@ import castalia.actors.{JsonEndpointActor, JsonResponseProviderEndpointActor, Js
 import castalia.matcher.RequestMatcherActor
 import castalia.matcher.RequestMatcherActor.{AddMatcher, FindMatch}
 import castalia.metrics.MetricsCollectorActor
+import castalia.model.Messages.{Done, EndpointMetricsGet, UpsertEndpoint, UpsertResponse}
+import castalia.model.Model.{StubConfig, StubResponse}
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/scala-academy/Castalia/file/2016772512/issues/source?bid=2998213&fileBranchId=3123962#l10)